Failed to kube_cover on Mac

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
bug

**Kubernetes version** (use `kubectl version`):
master (SHA 3371766d0afd8b8181b1aacf86e43bb4b3df4a4f)

**What happened**:
```
macpro:kubernetes klaus$ make check WHAT=plugin/pkg/scheduler/schedulercache/ KUBE_COVER=y
Running tests for APIVersion: v1,apps/v1beta1,authentication.k8s.io/v1beta1,authorization.k8s.io/v1beta1,autoscaling/v1,batch/v1,batch/v2alpha1,certificates.k8s.io/v1alpha1,extensions/v1beta1,imagepolicy.k8s.io/v1alpha1,policy/v1beta1,rbac.authorization.k8s.io/v1alpha1,storage.k8s.io/v1beta1,federation/v1beta1
+++ [0113 10:21:28] Saving coverage output in '/tmp/k8s_coverage/v1,apps/v1beta1,authentication.k8s.io/v1beta1,authorization.k8s.io/v1beta1,autoscaling/v1,batch/v1,batch/v2alpha1,certificates.k8s.io/v1alpha1,extensions/v1beta1,imagepolicy.k8s.io/v1alpha1,policy/v1beta1,rbac.authorization.k8s.io/v1alpha1,storage.k8s.io/v1beta1,federation/v1beta1/20170113-102128'
skipped	k8s.io/kubernetes/cmd/libs/go2idl/generator
skipped	k8s.io/kubernetes/vendor/k8s.io/client-go/1.4/rest
hack/make-rules/test.sh: line 259: _pkg: unbound variable
+++ [0113 10:21:28] Combined coverage report: /tmp/k8s_coverage/v1,apps/v1beta1,authentication.k8s.io/v1beta1,authorization.k8s.io/v1beta1,autoscaling/v1,batch/v1,batch/v2alpha1,certificates.k8s.io/v1alpha1,extensions/v1beta1,imagepolicy.k8s.io/v1alpha1,policy/v1beta1,rbac.authorization.k8s.io/v1alpha1,storage.k8s.io/v1beta1,federation/v1beta1/20170113-102128/combined-coverage.html
make: *** [check] Error 1
```

**What you expected to happen**:
Generate code coverage report on Mac

**How to reproduce it** (as minimally and precisely as possible):
```
make check WHAT=plugin/pkg/scheduler/schedulercache/ KUBE_COVER=y
```

--- a/hack/make-rules/test.sh
+++ b/hack/make-rules/test.sh
@@ -263,14 +263,14 @@ runTests() {
         ${KUBE_TIMEOUT} \
         -cover -covermode=\"${KUBE_COVERMODE}\" \
         -coverprofile=\"${cover_report_dir}/\${_pkg}/${cover_profile}\" \
-        \"${_pkg}\" \
+        \"\${_pkg}\" \
         ${testargs[@]:+${testargs[@]}}
       go test ${goflags[@]:+${goflags[@]}} \
         ${KUBE_RACE} \
         ${KUBE_TIMEOUT} \
         -cover -covermode=\"${KUBE_COVERMODE}\" \
         -coverprofile=\"${cover_report_dir}/\${_pkg}/${cover_profile}\" \
-        \"${_pkg}\" \
+        \"\${_pkg}\" \
         ${testargs[@]:+${testargs[@]}} \
       | tee ${junit_filename_prefix:+\"${junit_filename_prefix}-\$_pkg_out.stdout\"} \
       | grep \"${go_test_grep_pattern}\"" \

